config/locales/en/en.yml

```diff
@@ -341,6 +341,7 @@ en:
     misc: "There was an error communicating with QuickBooks. An administrator has been notified. Some data may be out of date. (Error: %{msg})"
     service_unavailable: "QuickBooks is temporarily unavailable. Please try again later."
     negative_balance: "This transaction is not allowed because it would result in a negative balance. The total balance on this date is %{amt}."
+  remember_me: "Remember Me"
```

Comment:
Does devise not have this in their own locale files? I'd check https://github.com/tigrish/devise-i18n if we aren't already using it, as it might save us some work in the future.

Reply:
This was what I found in their documentation

```
Assuming you have not previously generated Devise's views into your project, that's all you need to do. If you have previously done this, *you will need to regenerate your views and then reapply whatever customizations that made you do this in the first place.*
```

Do we still want to proceed with this?
@meltheadorable

Reply:
@arisah-ijeoma I'm honestly not sure what the best approach will be here. I'd like to "uncustomize" the views here and then push to the i18n gem if at all possible, because it will save us maintenance difficulties later, and I'm pretty sure that there are things like password reset emails and other stuff associated with devise that are completely untranslated instead of only partially untranslated. The more we can rely on the gem for this the better, but I don't know what customizations we have made to the base devise setup.